cpp/notebooks/cpp_pcl-1.10.ipynb

```diff
@@ -2,59 +2,81 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "80bc1cb4",
+   "execution_count": 4,
+   "id": "c8fba46e",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Files written to ./load_boost_git.h\n"
+     ]
+    }
+   ],
    "source": [
-    "! if [ ! -d boost_git ]; then git clone -b boost-1.71.0 --single-branch --recurse-submodules https://github.com/boostorg/boost.git boost_git; fi"
+    "! ../scripts/generate_cling_boost.py "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "c8fba46e",
+   "execution_count": 5,
+   "id": "11057ae0",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Files written to ./load_eigen3.h\n"
+     ]
+    }
+   ],
    "source": [
-    "! ../scripts/generate_cling_boost.py boost_git"
+    "! ../scripts/generate_cling_3rd_party.py eigen3"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "11057ae0",
+   "cell_type": "markdown",
+   "id": "a8d79273",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "! ../scripts/generate_cling_3rd_party.py eigen3"
+    "### Note if your are using Ubuntu 18 please change pcl_common-1.10 -> pcl_common-1.8"
```

Comment:
What about having another notebook?
Because the filename contains the version.
But, yeah it's good to have the note here

Reply:
Got it, added new file and added comments on both